Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -961,11 +961,13 @@ - (void)updatePopupLayout
                        [context.actionName isEqualToString:LPMT_HTML_NAME]);
     BOOL isWeb = [context.actionName isEqualToString:LPMT_WEB_INTERSTITIAL_NAME] ||
                  [context.actionName isEqualToString:LPMT_HTML_NAME];
-    
-    CGFloat statusBarHeight = ([[UIApplication sharedApplication] isStatusBarHidden] || !fullscreen) ? 0
+
+    UIEdgeInsets safeAreaInsets = [self safeAreaInsets];
+
+    CGFloat statusBarHeight = ([[UIApplication sharedApplication] isStatusBarHidden] || !fullscreen) ? safeAreaInsets.top
```

Comment:
I am not sure the use of "safeAreaInsets.top" here, because safeAreaInsets.top will change if status bar is hidden. 

Instead, can we consistently use the value 20, defined under a constant. This is consistent to your code below.
#define kTopInset 20.0

CGFloat statusBarHeight = ([[UIApplication sharedApplication] isStatusBarHidden] || !fullscreen) ? kTopInset